contracts/modules/LoanOpenings.sol

```diff
@@ -360,50 +361,58 @@ contract LoanOpenings is
             _initializeInterest(
                 loanParamsLocal,
                 loanLocal,
-                sentValues[0], /// newRate
-                sentValues[1], /// newPrincipal,
-                sentValues[2] /// torqueInterest
+                sentValues.interestRate, /// newRate
+                sentValues.newPrincipal, /// newPrincipal,
+                sentValues.interestInitialAmount /// torqueInterest
             );
 
         /// substract out interest from usable loanToken sent.
-        sentValues[3] = sentValues[3].sub(amount);
+        sentValues.loanTokenSent = sentValues.loanTokenSent.sub(amount);
 
         if (isTorqueLoan) {
-            require(sentValues[3] == 0, "surplus loan token");
+            require(sentValues.loanTokenSent == 0, "surplus loan token");
 
-            uint256 borrowingFee = _getBorrowingFee(sentValues[4]);
+            uint256 borrowingFee = _getBorrowingFee(sentValues.collateralTokenSent);
             // need to temp into local state to avoid
             address _collateralToken = loanParamsLocal.collateralToken;
             address _loanToken = loanParamsLocal.loanToken;
             if (borrowingFee != 0) {
                 _payBorrowingFee(
-                    sentAddresses[1], /// borrower
+                    sentAddresses.borrower, /// borrower
                     loanLocal.id,
                     _collateralToken, /// fee token
                     _loanToken, /// pairToken (used to check if there is any special rebates or not) -- to pay fee reward
                     borrowingFee
                 );
 
-                sentValues[4] = sentValues[4] /// collateralTokenReceived
+                sentValues.collateralTokenSent = sentValues
+                    .collateralTokenSent /// collateralTokenReceived
```

Comment:
pls consider adding a new var to the sentValues `collateralTokenReceived` or `sentAmountSubtraction`
which would add clarity at a price of another var in the struct.
separation of the amount sent and received (sent sub fees) this would lead to some code changes as not amountSent but amountReceived (or amountSent.sub(sentAmountSubtraction)) will be used further on

Reply:
I don't think adding variables here is efficient since the data is being used actually the same, it's just a matter of term which is different between the protocol modules (collateralTokenReceived) and the loan token logic (collateralTokenSent), but the value is the same for both.
So we can just unify the terms for both protocol and loan token logic

Reply:
@ororopickpocket wdyt?

Reply:
Will remove the comment